controllers/kogitoserverlessplatform_controller_test.go

```diff
@@ -37,23 +36,15 @@ func TestKogitoServerlessPlatformController(t *testing.T) {
 			namespace = "kogito-serverless-operator-system"
 		)
 		// Create a KogitoServerlessPlatform object with metadata and spec.
-		ksp, errYaml := utils.GetKogitoServerlessPlatform("../config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml")
-		if errYaml != nil {
-			t.Fatalf("Error reading YAML file #%v ", errYaml)
-		}
+		ksp := test.GetKogitoServerlessPlatform("../config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml")
```

Comment:
@davidesalerno shouldn't we use there the example that already have the kaniko task?